src/app/legacy/containers/Navigation/index.canonical.jsx

```diff
@@ -48,8 +60,10 @@ const CanonicalNavigationContainer = ({
             {scrollableListItems}
           </ScrollableNavigation>
         )}
+        <Divider />
```

Comment:
Should this only display if `experimentVariant` is a valid value? Appears to have caused some visual diffs.

Reply:
This was originally a border-bottom on the navigation component, and ended up being present underneath the scrollable promo which was against designs. It was in an ::after that I didn't seem to be able to move to being just under the navigation scroll. Making the styled divider allowed me to move it above the scrollable promo and underneath the scrollable nav. 

If we make it conditional on the experimentVariant being truthy we will also need to put back the original border bottom and make that conditional as well.

It may be possible to place onto the nav again in a different way. Or I can just fix the diff by making it full width, as the diff is just that it doesn't fill the width of the whole page.